mne/viz/_brain/_brain.py

```diff
@@ -2316,6 +2317,43 @@ def add_label(self, label, color=None, alpha=1, scalar_thresh=None,
             self._labels[hemi].append(label)
         self._renderer._update()
 
+    @fill_doc
+    def add_head(self, dense=True, color=None, alpha=0.5):
+        """Add a mesh to render the outer head surface.
+
+        Parameters
+        ----------
+        dense : bool
+            Whether to plot the dense head (``seghead``) or the less dense head
+            (``head``).
+        color : matplotlib-style color | None
+            A list of anything matplotlib accepts: string, RGB, hex, etc.
+            (default: "gray").
```

Comment:
just a note-to-self that we should consider unifying our "matplotlib-compatible color" docstrings, which are rather heterogeneous at the moment (`git grep matplotlib-` for a sampling).  But not necessary for this PR